pkg/sloop/queries/timerange.go

```diff
@@ -62,18 +62,18 @@ func computeTimeRangeInternal(params url.Values, endOfTime time.Time, maxLookBac
 		}
 	}
 
+	// always use customer input as computedEnd
+	computedStart, computedEnd, err = getTimeRangeFromStartEnd(startTimeVal, endTimeVal)
```

Comment:
@mengyaoyang11 we are breaking the existing behavior - if lookBack is passed then startTime and endTime will be empty and getTimeRangeFromStartEnd will return error.

Reply:
updated